controllers/storagecluster/reconcile.go

```diff
@@ -429,6 +429,7 @@ func (r *StorageClusterReconciler) reconcilePhases(
 		// for external cluster, we have a different set of ensure functions
 		// preserve list order
 		objs = []resourceManager{
+			&ocsProviderServer{},
```

Comment:
adding it here so that we deploy ocs-provider-server in external mode

Reply:
Why do we need to run this in external mode?

Reply:
I think we decided that provider API server will be running in all modes